.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
.editorconfig configures editors but does not enforce or do much in terms of code style. I use it on larger projects, but in this project we have `eslint`, which _does_ enforce code style.